spec/presenters/confirmation_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'spec_helper'
+require 'active_support/inflector'
```

Comment:
Is this required?

Reply:
Because this spec doesn't include `rails_helper`, if you run these specs in isolation you get an error because `ActiveSupport::Inflector` isn't loaded.

Reply:
Ah I see: this is because `Confirmation` directly references `AppointmentsPresenter` which uses `to_sentence`. I guess this is a sign that our test of Confirmation is actually integration testing the presenter classes as well. I don't think there's any real benefit to addressing that now though.